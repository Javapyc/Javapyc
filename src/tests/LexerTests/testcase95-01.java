/** This is really just to test for comments
    // Should be ignored **
    / public int i;
    */
System.out.println(1); //hoorah!