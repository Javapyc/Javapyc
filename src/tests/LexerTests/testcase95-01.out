ReservedWord, System.out.println
Delimiter, (
Integer, 1
Delimiter, )
Delimiter, ;
